server/pull_request.go

```diff
@@ -60,12 +66,11 @@ func (s *Server) handlePullRequestEvent(event *PullRequestEvent) {
 			s.removeLabel(mobileRepoOwner, mobileRepoName, pr.Number, s.Config.BuildMobileAppTag)
 		}
 
-		if pr.RepoName == s.Config.EnterpriseTriggerReponame {
-			if *event.Label.Name == s.Config.EnterpriseTriggerLabel {
-				mlog.Info("Label to run ee tests", mlog.Int("pr", event.PRNumber), mlog.String("repo", pr.RepoName))
-				go s.triggerEnterpriseTests(pr)
-				s.removeLabel(pr.RepoOwner, pr.RepoName, pr.Number, s.Config.EnterpriseTriggerLabel)
-			}
+		if pr.RepoName == s.Config.EnterpriseTriggerReponame &&
+			*event.Label.Name == s.Config.EnterpriseTriggerLabel {
+			mlog.Info("Label to run ee tests", mlog.Int("pr", event.PRNumber), mlog.String("repo", pr.RepoName))
+			go s.triggerEnterpriseTests(pr)
+			s.removeLabel(pr.RepoOwner, pr.RepoName, pr.Number, s.Config.EnterpriseTriggerLabel)
```

Comment:
why remove the label?

Reply:
I used the same approach as the Build mobile app and the spinmint test servers had. It serves as additional user feedback.